### DISCUSSION TITLE
How to get scroll in Sub Flow

### DISCUSSION THREAD

**Discussion by gopusrikanth04** [xyflow/xyflow#2088] [Q&A]
Hello Team,

I am using Sub Flows adding child nodes dynamically 

https://reactflow.dev/docs/guides/sub-flows/

I took fixed width and height of the group node . I display nodes in group node I need to get scroll when it is overflow.

How can I solve this. any help thank you 
 
![image](https://user-images.githubusercontent.com/47174113/163945023-372385c3-a492-4167-80f2-7bbc271d0c66.png)

**Comment by PoojaHoney** [comment]
Hello @bcakmakoglu

The issue was same for me, not resolved from this - [https://github.com/wbkd/react-flow/discussions/2085](https://github.com/wbkd/react-flow/discussions/2085) 

Need scrolling for the child items in sub flow, the child items were exceeding or overflowing from the sub flow. Could you please help me over this.

Thank you in advance.

**Reply by bcakmakoglu** [reply]
If I understand you correctly I believe you might want to use `panOnScroll` instead of using `overflow: scroll` on your list.
The reason for that is that overflow scroll might scroll your elements into the "background" but the handle positions are not calculated at a new position. Instead edges will then start to point seemingly nowhere when the Handle is scrolled away.

Child elements are also not contained inside the parent-node in the DOM, they're actually next to each other, so the markup looks like this
```markup
<div class="parent-node">
...
</div>

<div class="child-node">
...
</div>
```

Putting an `overflow: scroll` on the parent node would do nothing to hide the child in this case.

Instead use `<ReactFlow panOnScroll />` which will switch scroll behavior to panning the pane, meaning you will move up or downwards (or sideways on a track-pad but you can disable horizontal-panning with `panOnScrollMode={PanOnScrollMode.Vertical}`).
So instead of scrolling *inside* the node you would scroll along the graph.

Here's an [example](https://codesandbox.io/s/sandpack-project-forked-irhk0e?file=/App.js) of what I mean.

I hope this is more what you were looking for 😄

**Comment by hengxian-jiang** [comment]
Hi team, I have the similar use cases like the overflow problem here. Need to scroll the child item in subflow but seems like there no props or hooks can achieve this. Could you help me over this?

Thank you in advance.

**Comment by ibagov** [comment]
Hi everyone, any updates on this issue? I'm having the same problem currently and can't seem to find any non-hacky solution.
